### PR TITLE
[gSoap] Update to 2.8.102 and re-enable x64 Builds

### PR DESCRIPTION
**Pull request by xgcssch**
This PR updates the port to upstream version 2.8.102
Furthermore it re-enables the builds for x64, which were erroneously disabled

### CODE REVIEW

File: ports/gsoap/CONTROL [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Source: gsoap
-Version: 2.8.93-3
+Version: 2.8.102
 Build-Depends: curl
 Homepage: https://sourceforge.net/projects/gsoap2/
 Description: The gSOAP toolkit is a C and C++ software development toolkit for SOAP and REST XML Web services and generic C/C++ XML data bindings.
-Supports: x86 & windows
\ No newline at end of file
+Supports: x86 & x64 & windows
```
Comment (strega-nil) [inline ports/gsoap/CONTROL:6]: This is broken (and for some reason we don't catch it in CI; cc @ras0219-msft). This should be `(x86 | x64) & windows`

**Review by strega-nil — CHANGES_REQUESTED** [review]

File: ports/gsoap/CONTROL (hunk 1 above)
Comment (xgcssch) [reply ports/gsoap/CONTROL]: Performed the supposed change.

**Review by strega-nil — APPROVED** [review]

**Comment by strega-nil** [comment]
Alright, awesome! Thanks @xgcssch :)

**Pull request merged**